src/renderer/library/components/catalog/TagLayout copy.tsx

```diff
@@ -0,0 +1,100 @@
+// ==LICENSE-BEGIN==
```

Comment:
`TagLayout copy.tsx` is a wrong filename

Reply:
This file looks like a redundant / unused copy of the original `TagLayout.tsx`?

Reply:
fixed